Makefile

```diff
@@ -87,6 +87,11 @@ lint: generate vanity-import-check pprof-execs
 	$(MAKE) lint -C support/ebpf
 	docker run --rm -t -v $$(pwd):/app -w /app golangci/golangci-lint:$(GOLANGCI_LINT_VERSION) sh -c "golangci-lint version && golangci-lint config verify && golangci-lint run --max-issues-per-linter -1 --max-same-issues -1"
 
+format: format-ebpf
+	gofmt -w -s ./
+	go tool goimports -w ./
+	go tool gofumpt -w ./
```

Comment:
Nit: Let's add one more target for Go-only formatting (as the eBPF code seldom changes)

```suggestion
format: format-go format-ebpf

format-go:
	gofmt -w -s ./
	go tool goimports -w ./
	go tool gofumpt -w ./
```